### ISSUE TITLE
#\No-Break_Space missing in LispWorks

### ISSUE THREAD

**Issue by CK-DE** [vindarel/cl-str#119]
str.lisp doesn't compile with LispWorks 7.1 since there is no character called #\No-Break_Space. In LispWorks it is called #\No-Break-Space, i.e. a dash instead of an underline is used.

May I kindly ask you to use .(code-char 160) instead or use a reader conditional.

**Comment by vindarel** [comment]
Are you using Quicklisp or `str` from master?

A fix was merged but in February and Quicklisp is not up to date.

see https://github.com/vindarel/cl-str/pull/108/files (for https://github.com/vindarel/cl-str/issues/99)

**Comment by CK-DE** [comment]
Yes, I am using `cl-str` via Quicklisp. I will try to get the fixed version via Ultralisp or check it out directly from Github, if this doesn't work.

Thanks for your quick reply and sorry for bothering you with an already fixed bug.
